clients/payment/src/pages/PaymentPage.tsx

```diff
@@ -0,0 +1,5 @@
+import { PaymentForm } from "../components/PaymentForm";
+
+export const PaymentPage = () => {
+  return <PaymentForm />;
```

Comment:
I think `PaymentForm` is not super useful, you can put its content directly here.